src/main/java/figures/Trapezoid.java

```diff
@@ -0,0 +1,51 @@
+package figures;
+
+public class Trapezoid extends Figure {
+
+    private final int maxLongOfSide = 100;
```

Comment:
this is a constant field
```suggestion
    private static final int MAX_SIDE_LENGTH = 100;
```